doc/discussion/about_project.md

```diff
@@ -0,0 +1,29 @@
+About the QGreenland project
+============================
+
+Greenland-focused research, planning, and education crosses disciplinary
+boundaries. With QGreenland, a free data-viewing platform, anyone can explore a
+wide variety of Greenland-focused data identified by an `expert team and
+Editorial Board <https://qgreenland.org/our-team>`_.
```

Comment:
Not sure this and the following links are working correctly. Shouldn't it just be something like the following?

```
[expert team andEditorial Board](https://qgreenland.org/our-team)
```

Doesn't currently render how I would expect: https://qgreenland.readthedocs.io/en/latest/discussion/about_project.html